docs/apache-airflow-providers-google/sensors/cloud/gcs.rst

```diff
@@ -0,0 +1,75 @@
+ .. Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+ ..   http://www.apache.org/licenses/LICENSE-2.0
+
+ .. Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+
+
+Google Cloud Storage Sensors
+============================
+
+Cloud Storage allows world-wide storage and retrieval of any amount of data at any time. 
+You can use Cloud Storage for a range of scenarios including serving website content, 
+storing data for archival and disaster recovery, or distributing large data objects to users via direct download.
+
+.. contents::
+  :depth: 1
+  :local:
+
+Prerequisite Tasks
+^^^^^^^^^^^^^^^^^^
+
+.. include::/sensors/_partials/prerequisite_tasks.rst
+
+.. _howto/sensor:GCSObjectExistenceSensor:
+
+GCSObjectExistenceSensor
+------------------------
+
+Use the :class:`~airflow.providers.google.cloud.sensors.gcs.GCSObjectExistenceSensor` to wait (poll) for the existence of a file in Google Cloud Storage.
+
+.. exampleinclude:: /../../airflow/providers/google/cloud/example_dags/example_gcs_sensors.py
+    :language: python
+    :dedent: 4
+    :start-after: [START howto_sensor_object_exists_task]
+    :end-before: [END howto_sensor_object_exists_task]
+
+.. _howto/sensor:GCSObjectsWtihPrefixExistenceSensor:
+
+GCSObjectsWtihPrefixExistenceSensor
```

Comment:
I just looked - is this a typo in the sensor? I did look at the source code and thsi at least matches that code, but if this is a typo we should definitely fix it 😬  Or does Wtih stand for something?

Reply:
Yeah I noticed this - I'll raise a separate issue for this. Changing the operator name could impact those already using the operators so I'll reach out to Airflow community to see what's best practice for this :)

Reply:
Raised here: https://github.com/apache/airflow/issues/14097

Reply:
Looks like it's fixed yay community for being so speedy!